test/main.js

```diff
@@ -2133,7 +2133,6 @@ describe('node-fetch', () => {
 		});
 
 		it('should accept data uri 2', async () => {
-			// this is from wikipedia: https://en.wikipedia.org/wiki/Data_URI_scheme
```

Comment:
@Richienb why do you think this documentation comment is irrelevant? It's the source reference.

Reply:
A link to Wikipedia about data uris is not necessary since it might as well go in the readme.

Reply:
It's not only about data URIs, this specific URI for the test was copied from the article, so, any future maintainer will have no doubts that the actual URI is correct.